recipes/musdb/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "musdb" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 27aac71095f919c7e0cfb526af1e6f745275645e959f759c42711354ea0fcbf7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # this project need libsndfile, however, there is no such package for win platform
+  skip: true  # [win]
```

If it's noarch it's okay for windows to fail with missing dependencies:
```suggestion
  noarch: python
```

---

Let's check that way, the `stempeg` requirement will sure fail for Windows as it does not have a Windows version (@see https://github.com/conda-forge/staged-recipes/pull/9998).